### DISCUSSION TITLE
How Can I achieve Auto Routing like what is happening in Nuxtjs?

### DISCUSSION THREAD

**Discussion by JenuelDev** [vuejs/router#2199] [Help and Questions] [answered]
I just want to create routes based on my view folder, how do I do that? So that I dont have to edit the route.ts file whenever a new view is added. It would be hard to manage if their are too many views. Thanks in advance for anyone going to discuss on this one.

**Comment by posva** [comment, marked as answer]
https://github.com/posva/unplugin-vue-router
